man/fi_endpoint.3.md

```diff
@@ -919,6 +919,17 @@ wire protocols.  The following tag formats are defined:
   Applications that use the CCL format pass in the payload identifier
   directly as the tag and set ignore bits to 0.
 
+*FI_TAG_RPC*
+
+: The FI_TAG_RPC flag is used to indicate that tags are being utilized to match RPC
+  replies with requests. When specified via fi_getinfo, the caller ensures that
+  a receive buffer with a corresponding tag to receive an RPC reply has been posted
+  prior to sending an RPC request.  As a result, unexpected message handling may be
+  optimized.
+
+  When FI_TAG_RPC is specified, unmatched received messages should be identified
+  as stale replies and discarded.
```

Comment:
>  When FI_TAG_RPC is specified, unmatched received messages should be identified
  as stale replies and discarded.

I am not sure if stating 'unmatched receive messages should be identified as stale' is helpful. In addition, should adds ambiguity to whether or not a message is discarded. I think the following is clearer.

>When FI_TAG_RPC is specified, unmatched received messages must be discarded.

Reply:
Done